functions/index.js

```diff
@@ -469,6 +469,44 @@ module.exports.reloadsheetdata = functions.https.onRequest(async (req, res) => {
   res.status(200).send(html_snippets);
 });
 
+async function updateSheetWithGeocodes(country) {
+  const client = await getAuthorizedClient();
+  let data = {}
+
+  // Open relevant sheet
+  const sheets = google.sheets('v4');
+  const request = {
+    spreadsheetId: SHEETS[country],
+    range: 'Form Responses 1'
+  };
+  request.auth = client;
+
+  // Transition code as we rename the output sheet from
+  // Form Responses 1 to Combined.
```

Comment:
We should just be able to use combined in the new function and not worry about transitioning... in fact I think we've actually transitioned all sheets anyway.

Reply:
GB wasn't transitioned earlier tonight, but updated.  I'll verify that others are fine.